.github/workflows/build2.yml

```diff
@@ -89,19 +89,23 @@ jobs:
             - name: Windows msvc Debug
               os: windows-latest
               compiler_cc: msvc
-              compiler_coptions:
+              compiler_coptions: /Zi
```

Comment:
Personally I'm not a fan of having the common (as in "normal usage") configuration settings of the project only specified in CI-specific scripts because it makes other people wanting to build that project locally having to track the right parameters and copy paste  them manually into their configurations (probably with errors).

I recommend instead to write these configurations as options-files (see `--options-file` common option) located in either the root directory or somewhere else convenient (the `build/` directory?), one file per configuration[*] and then use them in this CI script. Instead of `compiler_cc` and `compiler_c/loptions` here you would have `configuration_options` set to one of these files and passed to `bdep init ... --options-file ${{matrix.config.configuration_options}}$`.

That way a user can also just `bdep init ... --option-file config-msvc-debug.options` for example and get the same configuration (with only the toolchain version potentially different).

[*] One technique I used successfully is to have common options in common option files which then were "included" in other more specific option files using `--option-file common.options`, as you can use flags in these files. That's how I used the same basic debug/release flags for clang, gcc and emcsripten in one of my projects.